.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -28,6 +28,24 @@ body:
       placeholder: "macOS 11.5"
     validations:
       required: true
+  - type: textarea
+    attributes:
+      label: "checkhealth telescope"
+      description: "Output of `:checkhealth telescope`"
```

maybe call to action here?
```suggestion
      description: "run `:checkhealth telescope` and paste output"
```

---

i wanna keep it identical with what we have for `nvim --version`